docs/src/routes.tsx

```diff
@@ -4,16 +4,21 @@ import { BrowserRouter, Redirect, Route, Switch } from 'react-router-dom'
 import ExternalExampleLayout from './components/ExternalExampleLayout'
 import DocsLayout from './components/DocsLayout'
 import DocsRoot from './components/DocsRoot'
+import DocsBehaviorRoot from './components/DocsBehaviorRoot'
 
 import Introduction from './views/Introduction'
 import PageNotFound from './views/PageNotFound'
 
+const behaviorMenuItems = require('docs/src/componentMenuBehaviors')
+const behaviorType = behaviorMenuItems[0].type
```

Comment:
The same as comment above

Reply:
I need to help with this how to solve it..